cloudinit/config/cc_ubuntu_advantage.py

```diff
@@ -69,6 +69,19 @@
           - fips
         """
         ),
+        dedent(
+            """\
+        # Set a http(s) proxy before attaching the machine to an
+        # Ubuntu Advantage support contract and enabling the FIPS service.
+        ubuntu_advantage:
+          token: <ua_contract_token>
+          config:
+            http_proxy: <http_proxy>
+            https_proxy: <https_proxy>
```

Comment:
Probably want to provide a fake url format to give a more concrete example. Eventually our unit tests will valid URI format.
Also let's add the other proxy examples
```suggestion
            https_proxy: 'https://some-proxy:8088'
            global_apt_https_proxy: 'http://some-global-apt-proxy:8088/'
            global_apt_http_proxy: 'https://some-global-a'pt-proxy:8088/'          
            ua_apt_http_proxy: 'http://10.0.10.10:3128'
            ua_apt_https_proxy: 'https://10.0.10.10:3128'
```